packages/mobile/src/components/now-playing-drawer/PlayBar.tsx

```diff
@@ -108,6 +111,7 @@ export const PlayBar = (props: PlayBarProps) => {
   const renderFavoriteButton = () => {
     return (
       <FavoriteButton
+        isDisabled={currentUser?.user_id === track?.owner_id}
```

Comment:
nice add, im starting to wonder if we should add "id" + "type" to favorite buttons and just let it select and do all the logic, cause i think we have so much duplicate code for "onPress" for favorite button, and then also have to add "disabled" everywhere lol. do you think it would be worth a ticket? another addition to it would be moving from lottie to rive.

Reply:
yea i agree. i've done this type of thing in other places too. and it also would make it easier if we want to update animations, add toasts, etc.

Reply:
cool! ill go ahead and add a low pri backlog item